### DISCUSSION TITLE
Possible Issue: Freezing video output streams

### DISCUSSION THREAD

**Discussion by LongLiveCHIEF** [WarmUpTill/SceneSwitcher#484] [General]
I've had advanced scene switcher running my last 2 streams out of a series of 5 I've done so far. 

Something I noticed as I went back and watched my live streams, is that the last 2 seem to have freezing problems with the video output, while audio is fine.

I haven't done _any_ debugging yet, so this could be entirely unrelated to scene switcher, but it's the only thing that changed about my setup for the last 2 streams, so figured I'd start a discussion on it, in case anyone has debugging ideas related to scene switcher I can try, (in addition to other debugging I'll be looking at).

The [CNC 3040](https://www.youtube.com/channel/UCaaQEc2KNmC1GRlv2piR0Rw/videos) series is the one I'm talking about, part 4 and part 5 utilize advanced scene switcher.

**Comment by nacalai** [comment]
Well, first step would be to upload the OBS log and use the analyzer to discover potentional problems. Did you try that?

**Comment by LongLiveCHIEF** [comment]
Just following up, this was a problem with my bitrate settings and nothing to do with Scene Switcher
